.gitignore

```diff
@@ -1,5 +1,7 @@
 # Cargo
 /target
 
-# IntelliJ IDEA
-/.idea
+# IDE Configurations
+**/.vs/
+**/.vscode/
```

Comment:
There already is VS Code configuration in the repository, which I'm actively using.
```suggestion
```